awx/api/views.py

```diff
@@ -3520,17 +3550,20 @@ def get(self, request, *args, **kwargs):
             data.update(messages)
         return Response(data)
 
-    def post(self, request, *args, **kwargs):
-        obj = self.get_object()
-        if not request.user.can_access(self.model, 'copy', obj):
-            raise PermissionDenied()
-        new_obj = obj.user_copy(request.user)
-        if request.user not in new_obj.admin_role:
-            new_obj.admin_role.members.add(request.user)
-        data = OrderedDict()
-        data.update(WorkflowJobTemplateSerializer(
-            new_obj, context=self.get_serializer_context()).to_representation(new_obj))
-        return Response(data, status=status.HTTP_201_CREATED)
+    @staticmethod
+    def deep_copy_permission_check_func(user, new_objs):
+        for obj in new_objs:
+            for field_name in obj._get_workflow_job_field_names():
```

Comment:
`_get_workflow_job_field_names` is for workflow jobs, but it looks like you're iterating over nodes (different model)

Reply:
No, this is a method of `WorkflowNodeBase`: https://github.com/ansible/awx/blob/devel/awx/main/models/workflow.py#L82